Switch to CurveEditor tab

<!-- NOTE! The information you specify will be publicly accessible. -->
### Expected behaviour

### Actual behaviour

### Steps to reproduce

<hr/>

<table>
<tr><td>Build time</td><td>2022-05-11T11:22:04.516620</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>0d4cec29d936b43320715b78987297244bc29589</td></tr>
<tr><td>Defold engine sha</td><td>287c945fab310c324493e08b191ee1b1538ef973</td></tr>
<tr><td>Defold version</td><td>1.3.2</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/organizations/defold/issues/?query=id%3A"b87ad801a0a74b57bbcc18b121c762b7"'>b87ad801a0a74b57bbcc18b121c762b7</a></td></tr>
<tr><td>GPU</td><td>NVIDIA GeForce GTX 1660 SUPER/PCIe/SSE2</td></tr>
<tr><td>GPU Driver</td><td>4.6.0 NVIDIA 512.59</td></tr>
<tr><td>Java version</td><td>11.0.1+13</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>OS name</td><td>Windows 10</td></tr>
<tr><td>OS version</td><td>10.0</td></tr>
<table>

java.lang.AssertionError: Assert failed: No such output, input, or property :render-mode exists for node 144115188075855881 of type editor.curve-view/CurveView
In production: #{}
beh

--- a/editor/src/clj/dynamo/graph.clj
+++ b/editor/src/clj/dynamo/graph.clj
@@ -924,12 +924,6 @@
    (swap! *the-system* is/invalidate-outputs outputs)
    nil))
 
-(defn invalidate-node-outputs!
-  [node-id]
-  (let [labels (-> (node-type* node-id)
-                   (in/output-labels))]
-    (invalidate-outputs! (map (partial vector node-id) labels))))
-
 (defn node-instance*?
   "Returns true if the node is a member of a given type, including
    supertypes."

--- a/editor/src/clj/editor/scene.clj
+++ b/editor/src/clj/editor/scene.clj
@@ -548,6 +548,7 @@
 
 (g/defnode SceneRenderer
   (property info-label Label (dynamic visible (g/constantly false)))
+  (property render-mode g/Keyword (default :normal))
 
   (input active-view g/NodeID)
   (input scene g/Any :substitute substitute-scene)
@@ -785,7 +786,6 @@
   (property viewport Region (default (g/constantly (types/->Region 0 0 0 0))))
   (property active-updatable-ids g/Any)
   (property play-mode g/Keyword)
-  (property render-mode g/Keyword)
   (property drawable GLAutoDrawable)
   (property picking-drawable GLAutoDrawable)
   (property async-copy-state g/Any)
@@ -1315,9 +1315,7 @@
     scene-view-pane))
 
 (defn- make-scene-view [scene-graph ^Parent parent opts]
-  (let [view-id (g/make-node! scene-graph SceneView
-                              :updatable-states {}
-                              :render-mode :normal)
+  (let [view-id (g/make-node! scene-graph SceneView :updatable-states {})
         scene-view-pane (make-scene-view-pane view-id opts)]
     (ui/children! parent [scene-view-pane])
     (ui/with-controls scene-view-pane [scene-view-info-label]

